packages/medusa/src/services/region.ts

```diff
@@ -389,11 +451,15 @@ class RegionService extends BaseService {
 
   /**
    * Retrieves a region by its id.
-   * @param {string} regionId - the id of the region to retrieve
-   * @param {object} config - configuration settings
-   * @return {Region} the region
+   *
+   * @param regionId - the id of the region to retrieve
+   * @param config - configuration settings
+   * @return the region
    */
-  async retrieve(regionId, config = {}) {
+  async retrieve(
+    regionId: string,
+    config: FindRegionConfig = {}
```

Comment:
suggestion: Since this is only an alias, should we privileged `FindConfig<Region>` instead